src/basics.md

```diff
@@ -1265,6 +1266,32 @@ fn main() {
     println!("Time elapsed in expensive_function() is: {:?}", duration);
 }
 ```
+[ex-convert-datetime]: #ex-convert-datetime
+<a name="ex-convert-datetime"></a>
+## Converting a local time to UTC time and vice versa
```

Comment:
Adding "timezone" to the title would go a long way in making it more discoverable